java/sage/EPG.java

```diff
@@ -624,19 +626,37 @@ public void run()
     // JS 8/21/2016: This is no longer used.
     //boolean[] didAdd = new boolean[1];
     boolean updateFinished = true;
+    //JUSJOKEN: 2025-02-19 add ability to run FULL Maintenance at a specific time each day
     while (alive)
     {
+
       try{
 
-        if (Sage.time() - wiz.getLastMaintenance() > MAINTENANCE_FREQ)
-          reqMaintenanceType = MaintenanceType.FULL;
+        if(Sage.getBoolean("wizard/scheduled_maintenance", false)){
+            if (Sage.DBG) System.out.println("EPG next scheduled maintenance offset is " + Sage.getInt("wizard/" + SCHEDULED_MAINTENANCE_OFFSET, 0));
+
+            nextScheuldedMaintenanceTime = getNextScheduledMaintenanceTime();
```

Comment:
nextScheuldedMaintenanceTime is misspelled

Reply:
fixed